#####自定义服务的格式
#
# ----------
# 客户端请求发送的数据
# ---  
# 服务器响应发送的数据
#
###################
int32 request_01
int32 request_02
---
#自定义服务器响应传输的数据
int32 response_01